index.js

```diff
@@ -25,6 +26,25 @@ module.exports = {
     ...readdirSync(join(__dirname, 'rules')).map(rule => join(__dirname, 'rules', rule))
   ],
   overrides: [
+    {
+      files: [
+        '*.ts'
+      ],
+      extends: ["plugin:@typescript-eslint/recommended"],
+      rules: {
```

Comment:
I'm certain there are many more rules, let's keep them explicitly stated here. We expect people to read this part of the code often, and we can't risk confusing them with multiple hops and sources of truth 😅

_Also, it guarantees more stable controls as we bump these plugin versions internally._